lua/starfall/examples/basic_moving_hologram.lua

```diff
@@ -0,0 +1,64 @@
+--@name Basic Moving Hologram
+--@author Name
+--@server
+
+-- Example showcasing basic ways of creating and manipulating a hologram, it's meant for absolute beginners
+-- Stuff like variables, function definitions, etc. are explained here: https://github.com/thegrb93/StarfallEx/wiki/Lua-Starfall-Crash-Course
+
+-- Functions from the `builtin` library can be called directly, the following gives us the chip entity
+local c = chip()
+
+-- To create a hologram, we make use of the `holograms` library and call it's `create` function
+-- The function can take up to 4 parameters: `position`, `angle`, `model` and optionally `scale` (not used here)
+-- `position`: The `getPos` method will return entity's world position as a vector object
+-- `angle`: The `Angle` constructor will give us an angle object with all axis set to 0, unless specified otherwise
+-- `model`: The string describing path to the model, you can use spawn menu item's context menu to obtain it, in this case it's a G-Man
+local holo = holograms.create(c:getPos(), Angle(), "models/gman_high.mdl")
+
+-- Since holograms are just entities, the entity methods can also be used on them
+-- Note how the Starfall's documentation only specifies `Entity.setColor` variant, but omits the unnecessary `Hologram.setColor`
+-- This function expects a color object, we can create one using `Color` which in turn expects R,G,B and optionally alpha values in range 0-255
+holo:setColor(Color(128,0,255))
+
+-- We can also set the hologram's velocity using the dedicated `setAngVel` method
+-- Note that the rotation of the hologram caused by this is controlled by the game engine and won't cost us any CPU time
+holo:setAngVel(Angle(0,100,0))
+
+-- Let's create a function which we will be responsible for updating our hologram
+local function update_hologram()
+	-- Grab current position of the hologram and add to it a newly constructed vector object with the `X` axis set to 1
+	-- Most vector operations like adding, multiplying, etc. will not modify the components, but instead create a new object
+	local target_position = holo:getPos() + Vector(1, 0, 0)
+	
+	-- Set the hologram's position to the previously calculated vector `target_position`
+	holo:setPos(target_position)
```

Comment:
Could setVel be used instead?

Reply:
It could, in most cases it should. But then I wouldn't have a SIMPLE way to showcase the `Tick`.

Reply:
I mean... could maybe spin the hologram in a circle? But then you'd have comments on how `sin` and shit works, instead of explaining how Starfall itself works.

Reply:
All right. I think it all looks gook